landscapes/single_objective.py

```diff
@@ -575,7 +575,7 @@ def cosine_mixture(x):
     Test Problems,” Journal of Global Optimization, vol. 31, pp. 635-672, 2005.
     '''
 
-    return -0.1 * sum([cos(5*pi*v) for v in x]) - sum([v**2 for v in x])
```

Comment:
Based on the reference link you shared as well as the original publication [[here](https://digitalassets.lib.berkeley.edu/sdtr/ucb/text/224.pdf)] it looks as though the current implementation is correct? Both instances feature the `-` subtraction.